tools/talis/README.md

```diff
@@ -200,3 +246,82 @@ Finally, remember to tear down the cloud instances. This should work first try,
 # tears down the network
 talis down
 ```
+
+## Running Talis inside of a DO droplet
+
+Create a new droplet
+
+- Recommended Size: 32GB RAM 16CPU
+- SSH Keys: Add your SSH key
+
+SSH into the Droplet
+
+```sh
+ssh root@YOUR_DROPLET_IP
+```
+
+Install Deps
+
+```sh
+# Install Go
+snap install go --channel=1.24/stable --classic
+
+# Install Docker
+apt install docker.io -y
+systemctl start docker
+usermod -aG docker $USER
+
+# Install misc tools
+apt install git curl jq -y
+```
+
+Set up Go env
+
+```sh
+echo 'export GOPATH="$HOME/go"' >> ~/.profile
+echo 'export GOBIN="$GOPATH/bin"' >> ~/.profile
+echo 'export PATH="$GOBIN:$PATH"' >> ~/.profile
+source ~/.profile
+```
+
+Clone and Build
+
+```sh
+# Clone celestia-app and cd into it
+git clone https://github.com/celestiaorg/celestia-app.git
+cd celestia-app
+
+# Build binaries (celestia, celestia-appd, txsim)
+make build-talis-bins
+
+# Install talis
+go install ./tools/talis/
+```
+
+Set env variables
+
+```sh
+export DIGITALOCEAN_TOKEN="your_api_token_here"
+export TALIS_SSH_KEY_PATH="~/.ssh/id_ed25519_no_passphrase"
```

Comment:
fwiw one could use scripts that cache the ssh key so that it only has to be unlocked once per 15mintues whatever time interval.

that's likely better practice